lib/bootloader_setup.pm

```diff
@@ -871,30 +871,32 @@ sub tianocore_http_boot {
 
 sub zkvm_add_disk {
     my ($svirt) = @_;
-    if (my $hdd = get_var('HDD_1')) {
-        my $basename = basename($hdd);
-        my $basedir  = svirt_host_basedir();
-        my $hdd_dir  = "$basedir/openqa/share/factory/hdd";
-        my $hdd_path = $svirt->get_cmd_output("find $hdd_dir -name $basename | head -n1 | tr -d '\n'");
-        die "Unable to find image $basename in $hdd_dir" unless $hdd_path;
-        diag("HDD path found: $hdd_path");
-        if (get_var('PATCHED_SYSTEM')) {
-            diag('in patched systems just load the patched image');
-            my $name        = $svirt->name;
-            my $patched_img = "$zkvm_img_path/$name" . "a.img";
-            $svirt->add_disk({file => $patched_img, dev_id => 'a'});
+    my $dev_id = 'a';
+    my $numdisks = get_var('NUMDISKS') || '1';
+
+    foreach my $n (1 .. $numdisks) {
```

Comment:
Just another hint just to apply some functional programming, we never use `$n` in the loop, so what we actually need are the letters.
So one could write:
```perl
foreach my $dev_id (map { chr((ord 'a') + $_) } (0 .. $num)) {
    print $dev_id;
}
```
Then we can avoid increments in each iteration. If that's more readable - not sure =)

Reply:
Sorry, missed `my $hdd = get_var('HDD_' . $n)`, nevermind =))